packages/realm-web-integration/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+	"compilerOptions": {
+		"strict": true,
+		"module": "CommonJS",
+		"target": "es5",
```

Comment:
Valid point - we can use the es6 target (as Webpack bundles this for us anyway) - as for the module, we're currently using mocha 5 (and I cannot upgrade because of another dependency) and Mocha [doesn't support es-modules natively until v7](https://github.com/mochajs/mocha/issues/3006#issuecomment-592945722).

I'm adding this to my list of things todo for the issue to be "completed" (i.e. to fix the dependency on mocha 5, upgrade mocha and change the "module" config parameter).

Reply:
👍